engine/updategraph/src/main/java/io/deephaven/engine/liveness/LivenessScopeStack.java

```diff
@@ -161,20 +161,41 @@ public void close() {
      * Perform a computation guarded by a new {@link LivenessScope} that is released before this method returns. The
      * result of the computation is managed by the enclosing {@link LivenessManager}, as determined by {@link #peek()}.
      *
+     * <p>
+     * Equivalent to {@code computeEnclosed(computation, shouldEnclose.getAsBoolean(), shouldManageResult)}.
+     *
      * @param computation The computation to perform. Will be invoked exactly once.
      * @param shouldEnclose Whether its actually necessary to use a new LivenessScope for the computation. Will be
      *        invoked exactly once.
      * @param shouldManageResult Whether its necessary to manage the result with the enclosing LivenessScope. Will be
      *        invoked exactly once.
      * @return The result of {@code computation.get()}
+     * @see #computeEnclosed(Supplier, boolean, Predicate)
      */
     public static <RESULT_TYPE extends LivenessReferent> RESULT_TYPE computeEnclosed(
```

Comment:
Do we want to deprecate the older overloads?
Do we want to add a flavor that expects a `DynamicNode` result and doesn't accept `shouldManageResult`?

Reply:
Deprecated older overloads.

I'd prefer to create a better solution once `LivenessReferent` knows itself whether it needs to be managed or not. Do we have a ticket for that?

Reply:
https://github.com/deephaven/deephaven-core/issues/4575